Tasks/VSBuild/VSBuild.ps1

```diff
@@ -105,18 +97,22 @@ else
     Write-Warning (Get-LocalizedString -Key "Unable to find script {0}" -ArgumentList $scriptLocation)
 }
 
-if(!$msBuildLocation)
+# Determine the MSBuild version.
+$msbuildVersion = $null;
+switch ("$vsVersion")
 {
-    if(Get-Command -Name "Get-MSBuildLocation" -ErrorAction SilentlyContinue)
```

Comment:
Does anyone know why this was here? Was this a compatibility thing? Can't we always just assume this cmdlet is imported? It might be better if I just leave this condition in the code since we're at the end of 84.